google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -274,6 +274,12 @@ cloud.apis.service.account.add.roles.progress.message=Adding roles to service ac
 cloud.apis.service.account.create.key.progress.message=Creating a service account key
 cloud.apis.service.account.download.key.progress.message=Downloading the service account key
 
+cloud.libraries.bom.inspection.group.name=Google Cloud Java Library BOM Inspections
+cloud.libraries.version.with.bom.inspection.display.name=Dependency Version with Bom Inspection
```

Comment:
Should we keep uppercase BOM as well here?

Reply:
yes, should be fully capitalized everywhere